js/react/components/nav/index.jsx

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types';
+
+function Nav(props) {
+	const { user, agency } = props;
+
+	console.log('render nav', user, agency);
+
+	return (
+		<div className="w-100">
+			<div className="navbar navbar-expand-md mb-0 p-1 bg-white">
+				<div className="container my-0">
+					<a className="navbar-brand ms-3 m-lg-0 p-0" href="/" rel="home">
+						<img src="/img/new-donate-gifts-logo-2.png" alt="logo" />
+					</a>
+					<div className="collapse navbar-collapse justify-content-end">
+						<div className="navbar nav">
+							<div className="nav-item">
+								<a className="nav-link" href="/wishcards">
+									Wish Cards
+								</a>
+							</div>
+							<div className="nav-item">
+								<a className="nav-link" href="/mission">
+									Mission
+								</a>
+							</div>
+							<div className="nav-item">
+								<a className="nav-link" href="/howto">
+									How To
+								</a>
+							</div>
+							<div className="nav-item">
+								<a className="nav-link" href="/community">
+									Community
+								</a>
+							</div>
+							{!user ? (
+								<div className="nav-item">
+									<a href="/login" className="nav-link">
+										Log In
+									</a>
+								</div>
+							) : user.userRole == 'partner' && agency && agency.isVerified ? (
+								<>
+									<div className="nav-item">
+										<a className="nav-link" href="/wishcards/me">
+											My Wishes
+										</a>
+									</div>
+									<div className="nav-item">
+										<a className="nav-link" href="/wishcards/create">
+											Create a Wish Card
+										</a>
+									</div>
+								</>
+							) : user.userRole == 'admin' ? (
+								<li className="nav-item">
+									<a className="nav-link" href="/admin/">
+										Admin Panel
+									</a>
+								</li>
+							) : (
+								<li className="nav-item">
+									<a className="nav-link" href="/profile">
+										Profile
+									</a>
+								</li>
+							)}
```

Comment:
is it possible to do this condition without nested ternary expressions?
it's really hard to read and understand in this way

Reply:
Maybe with explicit && conditionals. 
![Screenshot from 2023-08-09 14-39-22](https://github.com/donategifts/donategifts/assets/79138407/98438a2c-7944-4e37-b347-89f15bdb1584)